libs/sdk-ui-kit/src/typings/featureFlags.ts

```diff
@@ -0,0 +1,16 @@
+// (C) 2020-2021 GoodData Corporation
+
+/**
+ * @internal
+ */
+export interface IWorkspacePermissions {
```

Comment:
it is not featureFlags but permissions, check the file name. And as said above, not sure if such interface doesnt belong to the SPI

Reply:
I used from sdk-backend-spi instead of creating a new interface in sdk-ui-kit thanks